pulseapi/entries/views.py

```diff
@@ -21,8 +56,8 @@ def post_validate(request):
     nonce = False
 
     if request.data:
-        csrf_token = request.data['csrfmiddlewaretoken']
-        nonce = request.data['nonce']
+        csrf_token = request.data.get('csrfmiddlewaretoken', False)
```

Comment:
this was throwing key errors so I turned it into `get` rather than direct access to get a fallback value.